finality-aleph/src/lib.rs

```diff
@@ -37,16 +37,32 @@ enum Error {
     SendData,
 }
 
-pub fn peers_set_config() -> sc_network::config::NonDefaultSetConfig {
+pub fn peers_set_config(protocol: Option<new_network::Protocol>) -> sc_network::config::NonDefaultSetConfig {
+    let name = match protocol {
+        Some(ref p) => p.name(),
+        _ => network::ALEPH_PROTOCOL_NAME.into(),
+    };
+
     let mut config = sc_network::config::NonDefaultSetConfig::new(
-        network::ALEPH_PROTOCOL_NAME.into(),
+        name,
         // max_notification_size should be larger than the maximum possible honest message size (in bytes).
         // Max size of alert is UNIT_SIZE * MAX_UNITS_IN_ALERT ~ 100 * 5000 = 50000 bytes
         // Max size of parents response UNIT_SIZE * N_MEMBERS ~ 100 * N_MEMBERS
         // When adding other (large) message types we need to make sure this limit is fine.
         1024 * 1024,
     );
-    config.set_config = sc_network::config::SetConfig::default();
+
+    config.set_config = match protocol {
+        Some(new_network::Protocol::Validator) => {
+            sc_network::config::SetConfig {
+                in_peers: 25,
+                out_peers: 0,
```

Comment:
Why out peers are set to 0? it looks like it is the only difference between this config and `SetConfig::default()`

Reply:
https://www.notion.so/cardinalcryptography/Remaining-new-network-work-ba2009b5fceb439fb36e68a92e3ff67d#25d0ba3c46644d66936e5ea33e96ff63

Reply:
Maybe I will change something else letter, currently there is no way to test it, so I wanted only to set 0 outgoing slots

Reply:
There is no way to test what exactly?

Reply:
cool, this is good enough for me. I am leaving this discussion open for Damian

Reply:
How does node is working with other protocol/config. There need to be done some additional things to enable `new_network`